fcgl/point-system
# Add testing dependencies

## Problem
Don't have testing framework set up

## Solution
Implement testing framework and add one unit test

## Testing
1. Build and Run: `docker-compose up --build`
2. Confirm that the application built successfully. Application won't build if a test fails
